### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by albanelia0**

### CODE REVIEW

File: staff/albanelia0/js/challenges/scripts/Array.prototype.toRandomCase.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * 
+ * @example: 
+ * var array = ['albanelia'];
+ * toRandomCase(array) // -> AlBaNeLiA
+ * @param {array} 
+ * @throws if the input is not an array
+ * 
+ */
+  Array.prototype.toRandomCase = function() {
+    if (typeof this !== 'array') throw Error('input is not a array!');
```
Comment (Asix94) [inline staff/albanelia0/js/challenges/scripts/Array.prototype.toRandomCase.js]: typeof 'array' is not exist, is better is your case if (! Array.isArray(this))

File: staff/albanelia0/js/challenges/scripts/arrayMerger.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ * 
+ * var array1 = [1, 2, 3, 5];
+ * var array2 = [2, 30, 1, 4];
+ * console.log(arrayMerger(array1, array2)); / / return →[{ 3}, { 5}, 30, 4];
+ * 
+ * @throws if the parameters are not an array
+ */
+
+function arrayMerger(array1, array2) {
+  if (!(array1 instanceof array)) throw Error('parameter not valid');
```
Comment (Asix94) [inline staff/albanelia0/js/challenges/scripts/arrayMerger.js]: instanceof Array

File: staff/albanelia0/js/challenges/scripts/arrayMerger.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  if (!(array2 instanceof array)) throw Error('parameter not valid');
```
Comment (Asix94) [inline staff/albanelia0/js/challenges/scripts/arrayMerger.js]: instanceof Array

**Comment by manuelbarzi** [comment]
please, review previous comments and take action.

File: staff/albanelia0/js/challenges/scripts/Array.prototype.toRandomCase.js (hunk 1 above)
Comment (albanelia0) [reply staff/albanelia0/js/challenges/scripts/Array.prototype.toRandomCase.js]: Add changes! Thanks

File: staff/albanelia0/js/challenges/scripts/arrayMerger.js (hunk 2 above)
Comment (albanelia0) [reply staff/albanelia0/js/challenges/scripts/arrayMerger.js]: 👍

File: staff/albanelia0/js/challenges/specs/box.test.spec.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+'use stric';
```
Comment (agandia9) [inline staff/albanelia0/js/challenges/specs/box.test.spec.js:1]: strict*

**Pull request merged**